testing/MLDB-461_horizontal_ops_test.py

```diff
@@ -208,5 +207,29 @@ def test_horizontal_latest(self):
             ]
         )
 
-mldb.run_tests()
+    @unittest.expectedFailure
+    def test_mldbfb_558_no_need_to_cast_ts_on_data(self):
+        res = mldb.query("""
+            SELECT horizontal_min({'a', 'b'})
+        """)
+        self.assertEqual(res[1][1], 'a')
+
+        res = mldb.query("""
+            SELECT horizontal_max({'a', 'b'})
+        """)
+        self.assertEqual(res[1][1], 'b')
 
+        res = mldb.query("""
+            SELECT horizontal_min({'2015-01-01T00:00:00Z',
+                                   '2016-01-01T00:00:00Z'})
+        """)
+        self.assertEqual(res[1][1], '2015-01-01T00:00:00Z')
+
+        res = mldb.query("""
+            SELECT horizontal_max({CAST ('2015-01-01T00:00:00Z' AS TIMESTAMP),
```

Comment:
can we just use `TIMESTAMP '2015-01-01T00:00:00Z'` which is the preferred way of creating timestamps?